cartridges/int_klaviyo/cartridge/scripts/klaviyoATC.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+/* Script Modules */
+var app = require('*/cartridge/scripts/app');
+var recreateHelpers = require('*/cartridge/scripts/recreateCartHelpers');
+
+/* API Includes */
+var Logger = require('dw/system/Logger');
+
+
+function addProductToCart(decodedItems, cartObj) {
+    try {
+        var productList = decodedItems.length ? decodedItems : null;
+        var cart = cartObj;
+
+        if (cart.object.allProductLineItems.length) {
+            recreateHelpers.clearCart(cart.object)
+        }
+
+        var params = request.httpParameterMap;
+        var format = params.hasOwnProperty('format') && params.format.stringValue ? params.format.stringValue.toLowerCase() : '';
+        var newBonusDiscountLineItem;
+        var Product = app.getModel('Product');
+        var productOptionModel;
+        var productToAdd;
+        var template = 'checkout/cart/minicart';
+
+        if (params.source && params.source.stringValue === 'wishlist' && params.cartAction && params.cartAction.stringValue === 'update') {
+            app.getController('Wishlist').ReplaceProductListItem();
```

Comment:
I'm not really sure what this conditional and call to ReplaceProductListItem is doing here. Is there a use case for Klaviyo recreate cart that would include source and cartAction params for wishlist update?

Reply:
Thanks for this - I've removed that out as suggested and have tested with various encoded objects, etc. Functionality is consistent without this.